<?xml version="1.0" encoding="UTF-8"?><magicdraw><comment classType="Comment" humanType="Comment" icon="index_files/icon_comment_2051026490.jpg" id="_10_5_622020d_1132928315279_322170_9861"><annotatedElement humanName="Annotated Element" mode="s,e"><class classType="Class" humanType="Class" icon="index_files/icon_class_1555793073.jpg" name="RuntimeErrorException" refid="_10_5_622020d_1132928315279_962538_9859"/></annotatedElement><body humanName="Body" mode="s,e">When a &lt;CODE&gt;java.lang.Error&lt;/CODE&gt; occurs in the agent it should be caught and
re-thrown as a &lt;CODE&gt;RuntimeErrorException&lt;/CODE&gt;.

@since 1.5
</body><documentation humanName="Documentation"/><owner classType="Class" humanName="Owner" humanType="Class" icon="index_files/icon_class_1555793073.jpg" mode="s,e" name="RuntimeErrorException" refid="_10_5_622020d_1132928315279_962538_9859"/><owningElement classType="Class" humanName="Owning Element" humanType="Class" icon="index_files/icon_class_1555793073.jpg" name="RuntimeErrorException" refid="_10_5_622020d_1132928315279_962538_9859"/><TO_DO humanName="To Do" mode="s,e"/></comment></magicdraw>